src/main/java/com/spotinst/sdkjava/model/ApiAuditEventLogs.java

```diff
@@ -0,0 +1,178 @@
+package com.spotinst.sdkjava.model;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonInclude;
+
+import java.util.HashSet;
+import java.util.Set;
+
+/**
+ * Created by ramrutha on 13/02/2023.
+ */
+@JsonIgnoreProperties(ignoreUnknown = true)
+@JsonInclude(JsonInclude.Include.NON_NULL)
+public class ApiAuditEventLogs {
```

Comment:
Rename to ApiAuditLogEvents.
Move this file to C:\Github\spotinst-sdk-java\src\main\java\com\spotinst\sdkjava\model\api\admin\account

Reply:
Done